src/Tizen.NUI/src/internal/RenderTask.cs

```diff
@@ -73,108 +64,12 @@ public static RenderTask GetRenderTaskFromPtr(global::System.IntPtr cPtr)
             return ret;
         }
 
-        /// <since_tizen> 3 </since_tizen>
-        public class Property : global::System.IDisposable
+        internal class Property
```

Comment:
please make this HiddenAPI (not internal) ; please add [EditorBrowsable], [Obsolete] attributes like others done.